src/containers/edm/propertytypes/PropertyTypeDetailsContainer.js

```diff
@@ -103,6 +167,14 @@ class PropertyTypeDetailsContainer extends React.Component<Props> {
   }
 }
 
+function mapStateToProps(state :Map<*, *>) :Object {
+
+  return {
+    entityTypes: state.getIn(['edm', 'entityTypes', 'entityTypes'], List()),
+    entityTypesById: state.getIn(['edm', 'entityTypes', 'entityTypesById'], Map())
```

Comment:
After moving the logic above into this function, you won't need `entityTypesById`.